test/database_api_test.dart

```diff
@@ -230,6 +230,25 @@ void testDatabase(
         Room(id: '!testroom:example.com', client: Client('testclient')));
     expect(users.isEmpty, true);
   });
+  test('storeUsers', () async {
+    final room =
+        Room(id: '!testroom:example.com', client: Client('testclient'));
+    await database.storeUsers(
```

Comment:
add test cases:
- add multiple users.
- add the existed user.

Reply:
- I wrote a new func to handle store user and check existing user in the database. Add new task for this func pls review it ( `test/store_user_test.dart` ).


<img width="539" alt="Screenshot 2024-12-30 at 00 16 00" src="https://github.com/user-attachments/assets/5cdd9c40-a911-4af4-9cf6-7c325cbe3cee" />